content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2306,9 +2307,7 @@ The name of the `name` argument on the `@relation` attribute can be omitted (`re
 ```prisma
 @relation(name: "UserOnPost", references: [id])
 @relation("UserOnPost", references: [id])
-
 // or
-
```

Comment:
bit unfortunate change - but probably nothing we can do?

Reply:
@janpio we can add `// prettier-ignore` annotation, but it will ignore everything.